From abda5b90b780aa537902f843e53db1ee80639f8c Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Mon, 3 Aug 2020 17:44:21 +0700
Subject: [PATCH 445/475] ARM: dts: iwg20d-q7-common: Disable pci1 to support
 USB3.0

According to RZ/G1 H/W manual, USB3.0 use one USB2.0 PHY.
Thereforce, pci1 (USB1) need to be disabled in order to enable USB3.0.

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/iwg20d-q7-common.dtsi | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm/boot/dts/iwg20d-q7-common.dtsi b/arch/arm/boot/dts/iwg20d-q7-common.dtsi
index 854bcb3..8ab6d43a 100644
--- a/arch/arm/boot/dts/iwg20d-q7-common.dtsi
+++ b/arch/arm/boot/dts/iwg20d-q7-common.dtsi
@@ -263,12 +263,6 @@
 	pinctrl-names = "default";
 };
 
-&pci1 {
-	status = "okay";
-	pinctrl-0 = <&usb1_pins>;
-	pinctrl-names = "default";
-};
-
 &pcie_bus_clk {
 	clock-frequency = <100000000>;
 };
-- 
2.7.4

